preview/MsixCore/msixmgr/InstallUI.cpp

```diff
@@ -340,7 +342,22 @@ HRESULT UI::ParseInfoFromPackage()
         {
         case InstallUIAdd:
         {
-            RETURN_IF_FAILED(m_packageManager->GetMsixPackageInfo(m_path, m_packageInfo));
+            HRESULT hrGetMsixPackageInfo = m_packageManager->GetMsixPackageInfo(m_path, m_packageInfo, MSIX_VALIDATION_OPTION::MSIX_VALIDATION_OPTION_FULL);
+            if (hrGetMsixPackageInfo == static_cast<HRESULT>(MSIX::Error::MissingAppxSignatureP7X))

```

Comment:
>Error [](start = 67, length = 5)

is this the only error we can handle in this way?

Reply:
Modified to add other errors.